src/main/java/views/add_edit_category_views/EditCategoryV.java

```diff
@@ -0,0 +1,95 @@
+package views.add_edit_category_views;
+
+import entities.EntityException;
+import entities.SessionStorage;
+import use_cases.add_edit_category_use_case.CategoryOD;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+/**
+ * View class for the EditCategoryV that extends Component class and implements ActionListener interface.
+ * Creates a new controller that produces a Category_OD object.
+ */
+public class EditCategoryV extends Component implements ActionListener {
+    CategoryC controller;
+    JComboBox<String> categoryCombo;
+    JTextField nameInput;
+    JTextField budgetInput;
+    String selectedCategory;
+    int monthID;
+    SessionStorage currSession;
+
+    public EditCategoryV(CategoryC controller, String[] existingCategory, int monthID, SessionStorage currSession) {
```

Comment:
Wrote the JavaDoc below instead of above the method. Also needs the @param thingies